utility/servicer/module.go

```diff
@@ -228,8 +230,16 @@ func (s *servicer) validateRelayChainSupport(relayChain *coreTypes.Identifiable,
 	}
 	defer readCtx.Release() //nolint:errcheck // We only need to make sure the readCtx is released
 
-	// DISCUSS: should we update the GetServicer signature to take a string instead?
-	servicer, err := readCtx.GetServicer([]byte(s.config.Address), currentHeight)
+
+	// The servicer address needs to be passed to persistence module, which uses hex.EncodeToString to convert the byte array to string.
+	//	Therefore, the address needs to be decoded as a byte array before passing it to the persistence module.
+	servicerAddrBz, err := hex.DecodeString(s.config.Address)
+	if err != nil {
+		return fmt.Errorf("error decoding servicer address %s: %w", s.config.Address, err)
+	}
+
+	// DISCUSS: should we update the GetServicer signature to take a string instead? alternatively, we could use []byte as type of servicer address
```

Comment:
See the TECHDEBT in `persisten_module` - third bullet point.

![Screenshot 2023-07-03 at 7 32 09 PM](https://github.com/pokt-network/pocket/assets/1892194/b439bbdf-78e2-4301-a155-45e3bb78cc25)

We can remove this comment because when the interface e is updated, all the callers will change

Reply:
Thanks, fixed.